### PR TITLE
Add support for several languages

### PR DESCRIPTION
**Pull request by gregorkaplan**
Adds definitions for Ukrainian, Serbian, Norwegian and Danish.

### CODE REVIEW

**Comment by bramstein** [comment]
I'm not a native speaker in any of the languages, but the changes themselves look good.

File: data/uk [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+version: 2
+anglicized_name: Ukrainian
+native_name: українська
```
Comment (frankrolf) [inline data/uk]: Українська

File: data/uk [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+codepoints:
+- 171                       # «
+- 187                       # »
+- 668                       # ʜ
+- 1028                      # Є
+- !ruby/range 1030..1031    # І,Ї
+- !ruby/range 1040..1065    # А,Б,В,Г,Д,Е,Ж,З,И,Й,К,Л,М,Н,О,П,Р,С,Т,У,Ф,Х,Ц,Ч,Ш,Щ
+- 1068                      # Ь
+- !ruby/range 1070..1103    # Ю,Я,а,б,в,г,д,е,ж,з,и,й,к,л,м,н,о,п,р,с,т,у,ф,х,ц,ч,ш,щ,ъ,ы,ь,э,ю,я
+- 1105                      # ё
```
Comment (frankrolf) [inline data/uk]: Remove. Not Ukrainian. https://en.wikipedia.org/wiki/Yo_(Cyrillic)

File: data/uk [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+- 1108                      # є
+- !ruby/range 1110..1111    # і,ї
+- 1137                      # ѱ
```
Comment (frankrolf) [inline data/uk]: Remove. That’s Greek or something.

File: data/uk [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+version: 2
+anglicized_name: Ukrainian
+native_name: Українська
+codepoints:
+- 171                       # «
+- 187                       # »
+- 668                       # ʜ
```
Comment (frankrolf) [inline data/uk]: Remove. Not Cyrillic.

File: data/uk [hunk 5]
```diff
@@ -0,0 +1,18 @@
+---
+version: 2
+anglicized_name: Ukrainian
+native_name: Українська
+codepoints:
+- 171                       # «
+- 187                       # »
```
Comment (frankrolf) [inline data/uk]: Shouldn’t punctuation be part of any subset?

File: data/uk (hunk 2 above)
Comment (gregorkaplan) [inline data/uk]: So should this be U+0451, @frankrolf ?

Comment (gregorkaplan) [inline data/uk]: nvm, @frankrolf , removing.

File: data/be [hunk 6]
```diff
@@ -0,0 +1,81 @@
+---
+version: 2
+anglicized_name: Belarusian
+native_name: беларуская мова
```
Comment (frankrolf) [inline data/be]: native name: Беларуская мова
(capital letter)

**Pull request merged**